dGame/EntityManager.cpp

```diff
@@ -69,6 +69,10 @@ void EntityManager::Initialize() {
 	m_HardcoreUscoreEnemiesMultiplier = hcUscoreMult.empty() ? 2 : std::stoi(hcUscoreMult);
 	auto hcDropInv = Game::config->GetValue("hardcore_dropinventory_on_death");
 	m_HardcoreDropinventoryOnDeath = hcDropInv.empty() ? false : (hcDropInv == "1");
+
+	// If instance is not zero, then hardcore mode is disabled
+	// aka minigames and props
+	if (dZoneManager::Instance()->GetZoneID().GetInstanceID() != 0) m_HardcoreMode = false;
```

Comment:
Believe instance-ID is different from clone-ID

Reply:
ah yes it is.  Glanced right over that 😅